packages/osintgram/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osintgram
+pkgver=1.3.r5.g50d44fd
+pkgrel=1
+pkgdesc='OSINT tool offering an interactive shell to perform analysis on Instagram account of any users by its nickname.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/Datalux/Osintgram'
+license=('GPL3')
+depends=('python' 'python-instagram-private-api' 'python-requests' 'python-requests-toolbelt' 'python-geopy' 'python-gnureadline' 'python-prettytable')
+makedepends=('git')
+source=("git+https://github.com/Datalux/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -Dm 644 requirements.txt "$pkgdir/usr/share/$pkgname/requirements.txt"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md doc/*.md
+
+  rm -rf LICENSE *.md doc/*.md .gitignore
+
+  cp -a * "$pkgdir/usr/share/$pkgname/"
+
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+source /usr/share/$pkgname/venv/bin/activate
```

Comment:
despite the fact that all dependencies were added, traces of the previous venv setup are remaining

Reply:
The package seems to work correctly without that venv line. The only issue should be inside the bin file that we generate at the end of package(). Osintgram looks for username in `config/credentials.ini`. Our package will set this file in `/usr/share/osintgram/config`.

The problem is: if I run `osintgram` command in a directory without `config/credentials.ini` it won't find it and will ask always for username. So, it works only if we run it in a directory with `config/credentials.ini` directory/file. Need to find a way to force osintgram to refer always to `/usr/share/osintgram/config` directory.

Reply:
Yes unfortunately often we have to use `cd /usr/share/$pkgname` in wrappers because tools call relative files in a wrong manner.

Reply:
Y. I implemented it.